### PR TITLE
LPS-78881 Create podspec to publish on Cocoapods

### PR DESCRIPTION
**Pull request by allanragec**

### CODE REVIEW

File: analytics-client-ios/Source/Api/Impl/AnalyticsClientImpl.swift [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+    
+    internal func sendAnalytics(analyticsMessage: AnalyticsEventsMessage)
+		throws -> String {
+			
+		let encoder = JSONEncoder()
+		encoder.dateEncodingStrategy = .iso8601
+		let data = try! encoder.encode(analyticsMessage)
+		
+		let result = URLSession.sendPost(url: baseUrl, body: data)
```
Comment (victorlaerte) [inline analytics-client-ios/Source/Api/Impl/AnalyticsClientImpl.swift]: is it possible to extract result tuple in different objets? (Data?, URLResponse?, Error?)

File: analytics-client-ios/Source/Api/Impl/IndentityClientImpl.swift [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	
+	func getUserId(identityContextMessage: IdentityContextMessage)
+		throws -> String {
+			
+		let url = _getBaseURL(identityContextMessage.analyticsKey)
+			
+		let encoder = JSONEncoder()
+		let data = try! encoder.encode(identityContextMessage)
+		
+		let result = URLSession.sendPost(url: url, body: data)
```
Comment (victorlaerte) [inline analytics-client-ios/Source/Api/Impl/IndentityClientImpl.swift]: extract tuple

File: analytics-client-ios/Source/Model/Event.swift [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+import Foundation
+
+/**
+* @author Allan Melo
+*/
+internal class Event: Codable {
+	init(applicationId: String, eventId: String, build: ((Event) -> Void)? = nil) {
+        self.applicationId = applicationId
+		self.eventId = eventId
```
Comment (victorlaerte) [inline analytics-client-ios/Source/Model/Event.swift]: SF

**Comment by igormatos** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**